WalletWasabi/WalletWasabi.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Should this be part of this PR? It's just it's not entirely related to this PR.

Reply:
As far as I can remember this is required for packages.lock. Without this, we will some errors. It should not be removed.